.github/workflows/workflow-run-forge.yaml

```diff
@@ -228,3 +233,14 @@ jobs:
       # Print out whether the job was skipped.
       - run: echo "Skipping forge test!"
         if: ${{ inputs.SKIP_JOB }}
+
+    - name: Upload results
+      # Run this step even if the test step ahead fails
+      if: "!cancelled() && ${{ inputs.SEND_RESULTS_TO_TRUNK }}"
```

Comment:
These GHA conditions are always tricky but looks good to me lol, itrustyoudog.

Reply:
LOL yes this is already used on the unit test reporting, but thats why im testing in a stacked PR